server/seeds/demoSeed.js

```diff
@@ -0,0 +1,26 @@
+const dotenv = require('dotenv').config({path:__dirname+'/./../.env'})
+const mongoose = require("mongoose");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+
+const seedDB = async (req, res, next) => {
+  const salt = await bcrypt.genSalt(10);
+  saltedPassword = await bcrypt.hash(process.env.DEMO_USER_PASSWORD, salt);
```

Comment:
The User model has a pre-save hook that will hash the password for us, this isn't necessary to be here. So, we can just pass in the `DEMO_USER_PASSWORD` directly in the demoUser object.

Reply:
Got it - thanks! Updates in latest version include:
- Removed hashing from demoSeed.js
- Noticed in my old code that the pre-save hook wasn't running (I think because I was using User.insertMany(demoUser) instead of demoUser.save(). All updated now in latest version.